src/cli/check-update.ts

```diff
@@ -1,8 +1,9 @@
+import path from 'path'
 import readPkgUp            from 'read-pkg-up'
 import { UpdateNotifier }   from 'update-notifier'
 
 export function checkUpdate (): void {
-  readPkgUp({ cwd: __dirname })
+  readPkgUp({ cwd: path.resolve() })
```

Comment:
`path.resolve()` is not the right solution.

use 

```ts
import { dirname } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
```

See https://stackoverflow.com/a/50052194/1123955